raft_test.go

```diff
@@ -1856,7 +1856,7 @@ func TestRaft_ProtocolVersion_Upgrade_2_3(t *testing.T) {
 	// Use the new ID-based API to add the server with its ID.
 	future := c.Leader().AddVoter(c1.rafts[0].localID, c1.rafts[0].localAddr, 0, 1*time.Second)
 	if err := future.Error(); err != nil {
-		c.FailNowf("[ERR] err: %v", err)
+		c.FailNowf("err: %v", err)
```

Comment:
Are these test changed accidental? They’re kinda useful for gripping test output

Reply:
They are intentional. The hclog library should include the log level in the entry every time anyway; and `c.FailNowf` is just a wrapper around an hclog call now. So it will include `ERR` or `ERROR` in square brackets for those entries.